docker-compose-config-a.yaml

```diff
@@ -347,9 +347,9 @@ services:
       logging: "config-a"
     command:
       [
-        "/bin/bash",
+        "/bin/sh",
```

Comment:
Does this provide any improvement in processing?

Reply:
I had to change it because apparently there's no support for /bin/bash in `alpine` images